### PR TITLE
disk monitor: dd diskUUID parameter in DiskCreate

### PR DESCRIPTION
**Pull request by derekbit**
If a disk is added before, the diskUUID is recorded in node.status.diskStatus. The recorded UUID can be used for validating if the disk is an expected one.

depends on: https://github.com/longhorn/longhorn-spdk-engine/pull/53 and https://github.com/longhorn/longhorn-instance-manager/pull/248

https://github.com/longhorn/longhorn/issues/6125

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -196,3 +196,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/longhorn/longhorn-instance-manager v1.4.0-rc1.0.20230613094429-85c7d1d7f70b => github.com/derekbit/longhorn-instance-manager v0.0.0-20230614171210-9aa89d527ac6
```
Comment (PhanLe1010) [inline go.mod:200]: Should remove this replace before merging?

Comment (derekbit) [reply go.mod:200]: Yes, waiting for the merge of https://github.com/longhorn/longhorn-spdk-engine/pull/53 and https://github.com/longhorn/longhorn-instance-manager/pull/248

**Review by innobead — COMMENTED** [review]
LGTM, just pending for dep update.

**Comment by derekbit** [comment]
Dependencies are updated.

**Pull request merged**